cmd/diff.go

```diff
@@ -65,6 +65,8 @@ that will be created, updated, or deleted.
 			"and exit code 1 if an error occurs.")
 	diffCmd.Flags().BoolVar(&dumpConfig.SkipCACerts, "skip-ca-certificates",
 		false, "do not diff CA certificates.")
+	diffCmd.Flags().BoolVar(&dumpConfig.SkipLicenses, "skip-licenses",
+		true, "do not diff licenses.")
```

Comment:
I think `skip-licenses` and `"do not diff licenses."` is a bit confusing (one is "positive", the other is "negative").

Should we do something like this instead:

`diffCmd.Flags().BoolVar(&dumpConfig.SkipLicenses, "no-skip-licenses", false, "do not diff licenses.")`

It feels more natural adding `--no-skip-licenses` (or similar) instead of `--skip-licenses=false`.

WDYT?

Reply:
In general, consistent language across like features is good UX. If we're adding a new switch that is similar to existing ones, it should normally follow the established pattern. Were we to change this, we should do so across the board and change the like flags for CA certificates and consumers.

I don't share the same confusion over both needing to be positive, but in either case, we should not change existing flags on a whim. Doing so would break any existing user scripts that use the existing flags, so I'd only recommend changing them if we have a documented history of user reports indicating that they failed to find the correct flag when attempting to skip these resources.

Changing the docstring would be a non-breaking change, but ultimately it's out of scope here: this PR seeks to add licenses and handle them similarly to other resources that can be excluded from actions, not to revise existing UX patterns. Please create a follow-up issue if you wish to change this.

Reply:
The difference between this case and the existing ones is the default value and the resulting usage of these flags.

For CA certificates, the default behaviour is to _include_ them into the computation. If you want to skip them, then you add `--skip-ca-certificates`.

For Licenses, the default behaviour is to _exclude_ them. With the current logic, if you want to include them, then you add `--skip-licenses=false`.

I don't think we should change any existing UX otherwise we are going to break existing usage and introduce unnecessary friction, so no follow-up issue will be created, but I do think that we should provide a good UX for new features. IMHO, either `--include-licenses` or `--no-skip-licenses` are more in line with this goal.